src/repr/type.ml

```diff
@@ -19,6 +19,9 @@ include Type_core
 include Staging
 open Utils
 
+type 'a encode_json = 'a Encode_json.t
+type 'a decode_json = 'a Decode_json.t
```

Comment:
I don't think you need to override these type here

Reply:
Indeed, thanks!